framework/include/controls/Control.h

```diff
@@ -114,11 +114,11 @@ class Control :
   void setControllableValueByName(const std::string & tag, const std::string & param_name, const T & value, bool warn_when_values_differ = false);
   ///@}
 
-private:
```

Comment:
I think we can just add a member function to make that call for us, I envision that this function will be needed by non-controls as well. At that time I think we will then make a ControlInterface class to provided access to inspect (const) parameters and connect them, but leave the setting to the actual Control object.

If I recall, you calling `addControllableParameterConnection` in other places as well, correct? So, maybe we need to make this interface now.